test/hca/helpers.unit.spec.jsx

```diff
@@ -38,7 +38,35 @@ describe('HCA helpers', () => {
         }
       };
 
-      expect(expensesLessThanIncome(formData)).to.be.true;
+      expect(expensesLessThanIncome('deductibleEducationExpenses')(formData)).to.be.true;
+    });
+    it('should show only if last non-zero expense amount', () => {
+      const formData = {
+        dependents: [{
+          grossIncome: 3
+        }],
+        deductibleEducationExpenses: 0,
+        deductibleFuneralExpenses: 0,
+        deductibleMedicalExpenses: 4
+      };
+
+      expect(expensesLessThanIncome('deductibleMedicalExpenses')(formData)).to.be.false;
+      expect(expensesLessThanIncome('deductibleEducationExpenses')(formData)).to.be.true;
+      expect(expensesLessThanIncome('deductibleFuneralExpenses')(formData)).to.be.true;
+    });
+    it('should shown only on last field if all expenses are filled', () => {
```

Comment:
Should we also remove this test as well?

Reply:
I think the test is still useful.

Reply:
Perhaps it could be rephrased? Show only if all expenses are filled?